pipenv/cli.py

```diff
@@ -1958,10 +1958,10 @@ def graph(bare=False, json=False):
         )
         sys.exit(1)
 
-    if json:
+    if is_json:
```

Comment:
What's the rationale for adding the extra letters to the flag? It seems like just json should be sufficiently expressive.